rtos/mbed_lib.json

```diff
@@ -21,6 +21,10 @@
          "idle-thread-stack-size-tickless-extra": {
             "help": "Additional size to add to the idle thread when tickless is enabled and LPTICKER_DELAY_TICKS is used",
             "value": 256
+         },
+         "idle-thread-stack-size-debug-extra": {
+            "help": "Additional size to add to the idle thread when tickless is enabled and LPTICKER_DELAY_TICKS is used",
+            "value": 256
```

Comment:
Some low end devices might fail because of limited memory, shall we keep this as 0 as default and update it to 256 for all ST targets only?

Reply:
Why only ST targets is affected to debug memory increase ?

Reply:
ST hal implementation uses more stack

Reply:
OK, I will propose default value 0,
and update few ST targets.

See quick measurements in #9106